src/pages/main/Main.js

```diff
@@ -1,7 +1,82 @@
 import React from 'react';
+import { BsCart3 } from 'react-icons/bs';
+import './Main.scss';
 
 const Main = () => {
-  return <div>Main</div>;
+  return (
+    <div className="main">
+      <div className="mainImg">
+        <img src="/images/grill.jpg"></img>
```

Comment:
현재 브랜치에선 이미지들이 안뜨는데 의도하신 게 맞을까요?


```suggestion
        <img src="/images/grill.jpg" />
```
내용이 없는 태그는 항상 바로 닫아주세요!

Reply:
반영 후 수정하였습니다!